catalog.html

```diff
@@ -0,0 +1,274 @@
+      </li>
+      <li class="connect-button">
+        <button class="connect" type="button">Connect</button>
+      </li>
+    </ul>
+  </header>
+  <main class="content">
+    <div class="banner">
+        <h2 class="banner__header">Explore catalog</h2>
+        <ul class="banner__navigation">
+            <li class="banner__nav-item"><a href="index.html" class="nav-link">Home</a></li>
+            <li class="banner__nav-item"><a href="" class="nav-link card--nonactive">Catalog</a></li>
+        </ul>
+    </div>
+    <section class="catalog__main-section">
+        <form action="" method="post" class="left-block-filters">
+            <div class="clothing">
+                <h3 class="clothing__header">Clothing</h3>
+                <div class="clothing__options">
+                    <label class="options"><input type="checkbox" name="clothing" value="combatant">Combatant</label>
+                    <label class="options"><input type="checkbox" name="clothing" value="ninja">Ninja</label>
+                    <label class="options"><input type="checkbox" name="clothing" value="king">King</label>
+                    <label class="options"><input type="checkbox" name="clothing" value="queen">Queen</label>
+                    <label class="options"><input type="checkbox" name="clothing" value="robot">Robot</label>
+                </div>
+            </div>
+            <div class="eyes">
+                <h3 class="eyes__header">Eyes</h3>
+                <div class="eyes__options">
+                    <label class="options"><input type="radio" name="eyes" value="yellow">Yellow</label>
+                    <label class="options"><input type="radio" name="eyes" value="green">Green</label>
+                    <label class="options"><input type="radio" name="eyes" value="mix">Mix</label>
+                    <label class="options"><input type="radio" name="eyes" value="orange">Orange</label>
+                </div>
+            </div>
+            <div class="face">
+                <h3 class="face__header">Face</h3>
+            </div>
+            <div class="skin">
+                <h3 class="skin__header">Skin</h3>
+            </div>
+            <div class="background">
+                <h3 class="background__header">Background</h3>
+            </div>
+            <div class="special">
+                <h3 class="special__header">Special</h3>
+                <div class="special__options">
+                    <label class="options"><input type="checkbox" name="special" value="yellow">Yellow</label>
+                    <label class="options"><input type="checkbox" name="special" value="green">Green</label>
+                    <label class="options"><input type="checkbox" name="special" value="mix">Mix</label>
+                    <label class="options"><input type="checkbox" name="special" value="orange">Orange</label>
+                </div>
+            </div>
+            <button class="apply-button" type="button">Apply</button>
```

Comment:
@Alinchik1 if this button is supposed to send the form data to the server, then it requires type submit

Reply:
changed to submit